diff_drive_controller/src/diff_drive_controller_parameter.yaml

```diff
@@ -0,0 +1,177 @@
+    description: "number of wheels on each side of robot body",
+  }
+  wheel_radius: {
+    type: double,
+    default_value: 0.0,
+    description: "assumed to be the same for both wheels",
+  }
+  wheel_separation_multiplier: {
+    type: double,
+    default_value: 1.0,
+    description: "",
+  }
+  left_wheel_radius_multiplier: {
+    type: double,
+    default_value: 1.0,
+    description: "",
+  }
+  right_wheel_radius_multiplier: {
+    type: double,
+    default_value: 1.0,
+    description: "",
+  }
+  odom_frame_id: {
+    type: string,
+    default_value: "odom",
+    description:  "frame id for odom",
+  }
+  base_frame_id: {
+    type: string,
+    default_value: "odom",
+    description: "frame id for base link",
+  }
+  pose_covariance_diagonal: {
+    type: double_array,
+    default_value: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+    description: "",
+  }
+  twist_covariance_diagonal: {
+    type: double_array,
+    default_value: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+    description: "",
+  }
+  open_loop: {
+    type: bool,
+    default_value: false,
+    description: "control in open loop mode",
+  }
+  position_feedback: {
+    type: bool,
+    default_value: true,
+    description: "",
+  }
+  enable_odom_tf: {
+    type: bool,
+    default_value: true,
+    description: "",
+  }
+  cmd_vel_timeout: {
+    type: double,
+    default_value: 0.5, # seconds
+    description: "Timeout to consider cmd_vel commands old",
+  }
+  publish_limited_velocity: {
+    type: bool,
+    default_value: false,
+      description: "Enforces velocity limit on published values",
+  }
+  velocity_rolling_window_size: {
+    type: int,
+    default_value: 10
+  }
+  use_stamped_vel: {
+    type: bool,
+    default_value: true
+  }
+  publish_rate: {
+    type: double,
+    default_value: 50.0 # Hz
+  }
+  linear:
+    x:
+      has_velocity_limits: {
+        type: bool,
+        default_value: false,
+      }
+      has_acceleration_limits: {
+        type: bool,
+        default_value: false,
+      }
+      has_jerk_limits: {
+        type: bool,
+        default_value: false,
+      }
+      max_velocity: {
+        type: double,
+        default_value: .NAN,
+      }
+      min_velocity: {
+        type: double,
+        default_value: .NAN,
+      }
+      max_acceleration: {
+        type: double,
+        default_value: .NAN,
+      }
+      min_acceleration: {
+        type: double,
+        default_value: .NAN,
+      }
+      max_jerk: {
+        type: double,
+        default_value: .NAN,
+      }
+      min_jerk: {
+        type: double,
+        default_value: .NAN,
+      }
+  angular:
+    z:
+      has_velocity_limits: {
+        type: bool,
+        default_value: false,
+      }
+      has_acceleration_limits: {
+        type: bool,
+        default_value: false,
+      }
+      has_jerk_limits: {
+        type: bool,
+        default_value: false,
+      }
+      max_velocity: {
+        type: double,
+        default_value: .NAN,
```

Comment:
NAN in a parameter breaks ROS2 python handling of parameters so commands like `ros2 param dump` stop working and throw a python exception. We probably should not use it as a default value.